decoimpact/business/entities/rules/string_parser_utils.py

```diff
@@ -76,30 +76,33 @@ def type_of_classification(class_val) -> str:
     """
 
```

Comment:
This might be implemented as a loop over pairs of strings. The list is not that long , but still.

Reply:
I tried, but it did not improve the readability